src/components/UI/Display/Tooltip/TooltipContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useRef, useState } from 'react';
+
+interface ITooltipContainerProps {
+  content: React.ReactNode;
+  target?: React.RefObject<HTMLElement>;
+}
+
+const TooltipContainer: React.FC<
+  React.PropsWithChildren<ITooltipContainerProps>
+> = ({ content, target, children }) => {
+  const [tooltipVisible, setTooltipVisible] = useState(false);
+  const createdTargetRef = useRef<HTMLElement>(null);
+
+  const tooltipTargetRef = target ?? createdTargetRef;
```

Comment:
If a target element is specified, the tooltip will align to that. Otherwise, it will align to the `children` elements